apps/web/public/static/locales/en/common.json

```diff
@@ -485,15 +478,11 @@
   "booking_confirmation": "Confirm your {{eventTypeTitle}} with {{profileName}}",
   "booking_reschedule_confirmation": "Reschedule your {{eventTypeTitle}} with {{profileName}}",
   "in_person_meeting": "In-person meeting",
-  "attendee_in_person": "In Person (Attendee Address)",
   "in_person": "In Person (Organizer Address)",
   "link_meeting": "Link meeting",
-  "phone_call": "Attendee Phone Number",
-  "your_number": "Your phone number",
   "phone_number": "Phone Number",
   "attendee_phone_number": "Attendee Phone Number",
   "organizer_phone_number": "Organizer Phone Number",
-  "host_phone_number": "Your Phone Number",
```

Comment:
Is it possible that the text is stored in DB and we won't find it in code search so we consider it unused?

Though if that is being done, we should fix that. But just wanted to put this question out there.